lib/palindrome_check.rb

```diff
@@ -1,5 +1,31 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
 def palindrome_check(my_phrase)
```

Comment:
I didn't see any comments on the time and space complexity of your algorithm. Can you think through it and explain it?
Gentle reminder: Always explain what _n_ stands for while explaining your time and space complexity. That along with your explanation for reasoning behind your answer, is the complete answer for the time and space complexity.